import logging
def test():
    try:
        do something
    except Exception as e:
        logging.warning(e)
    return